File: JAVA/src/main/java/com/inf_112/trabalho_final/model/Kimono.java
package com.inf_112.trabalho_final.model;

import lombok.Data;
import lombok.EqualsAndHashCode;

@Data
@EqualsAndHashCode(callSuper = true)
public class Kimono extends Produto {
    public enum Forma {
        TRADICIONAL, SLIM, ULTRASLIM
    }
   
    private double gramaturaTecido;
    private int encolhimento;
    private Forma forma;
}

File: APP/src/app/model/kimono.ts
import { Produto } from "./produto";

enum Forma {
    TRADICIONAL, SLIM, ULTRASLIM
}

export class Kimono extends Produto {
    gramaturaTecido: number;
    encolhimento: number;
    forma: Forma | null;

    constructor() {
        super();
        this.gramaturaTecido = 0;
        this.encolhimento = 0;
        this.forma = null; 0
    }
}
